docs/demos/dev-select/view.html

```diff
@@ -1,5 +1,15 @@
 <div ng-controller="DevSelectCtrl">
-  <a href="#" editable-select="user.status" e-multiple e-ng-options="s.value as s.text for s in statuses">
```

Comment:
Yes, it's not necessary, it's a habit of mine.  I can remove them if you like.

Angular does list it in their best practices section on directives:
https://docs.angularjs.org/guide/directive

An overview of the data attribute
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes